components/CommunityAnnouncements/CommunityAnnouncementListings.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="subpage">
+    <div v-for="(item, index) in items" :key="index">
+      <community-announcement-item :item="item" />
+      <div v-if="index !== items.length - 1 || bottomLink" class="seperator-path" />
+    </div>
+    <nuxt-link
+      v-if="bottomLink"
+      class="community-link mt-16"
+      :to="{
+        name: linkLocation
+      }"
+    >
+      {{ linkText }} &gt;
+    </nuxt-link>
+  </div>
+</template>
+
+<script>
+import CommunityAnnouncementItem from './CommunityAnnouncementItem.vue'
+
+export default {
+  name: 'CommunityAnnouncementListings',
+  components: {
+    CommunityAnnouncementItem
+  },
+  props: {
+    items: {
+      type: Array,
+      default: () => []
+    },
+    bottomLink: {
+      type: Boolean,
+      default: false
+    },
+    linkLocation: {
+      type: String,
+      default: 'news-and-events-community-spotlight'
+    },
+    linkText: {
+      type: String,
+      default: 'View All Community Announcements'
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/_variables.scss';
+
+.seperator-path {
+  width: 100%;
+  height: 2px;
+  background: rgb(216, 216, 216);
+  border-radius: 0px;
+  margin-bottom: 32px;
+  margin-top: 32px;
+}
+
+.subpage {
+  margin-bottom: 20px;
```

Comment:
You should be able to use a spacing class from the design system here: https://nih-sparc.github.io/sparc-design-system-components/?path=/docs/layout-spacing--spacing-values#margin

So in this case you can just use either mb-16 or mb-24